MovieApp/MovieApp/App/Repository/Model/MovieRepoModel.swift

```diff
@@ -13,12 +13,13 @@ struct MovieRepoModel {
     let genreIds: [Int]?
     let budget: Int?
     let runtime: Int?
+    var isFavorite: Bool
     
 }
 
 extension MovieRepoModel {
     
-    init(fromModel model: MovieDataModel) {
+    init(fromModel model: MovieDataModel, isFavorite favorite: Bool = true) {
```

Comment:
You can just have `isFavorite`, there's no need for param name.

Reply:
[Here](https://github.com/lihter/Movie-App/pull/38/commits/cd9b26a2e35cb7418ace1f07855e235f95125719#diff-16ec28a5469846b8ad26272a1925c9f70fa4bf8572c2a83c4b764c9caa874313R22)